insta/src/lib.rs

```diff
@@ -77,10 +77,10 @@
 //! This crate exports multiple macros for snapshot testing:
 //!
 //! - [`assert_snapshot!`] for comparing basic snapshots of
-//!   [`Display`](std::fmt::Display) outputs, often strings.
+//!   [`std::fmt::Display`] outputs, often strings.
 //! - [`assert_debug_snapshot!`] for comparing [`Debug`] outputs of values.
 //!
-//! The following macros require the use of serde's [`Serialize`](serde::Serialize):
+//! The following macros require the use of [`serde`]'s [`serde::Serialize`]:
```

Comment:
Same here; though could do

```suggestion
//! The following macros require the use of [`serde::Serialize`]:
```